ldoc/parse.lua

```diff
@@ -2,7 +2,7 @@
 
 local utils = require 'pl.utils'
 local List = require 'pl.List'
-local Map = require 'pl.Map'
+-- local Map = require 'pl.Map'
```

Comment:
left over?

Reply:
I commented rather than removing this because there was some commented code that used it (L161). It wasn't clear to me why the code was commented, so I figured I'd leave the require in the same condition in case somebody figures out using `Map(tags)` instead of `tags` is actually where this was headed instead of where it came from.